src/js/components/Beacon/BeaconQueryUi.js

```diff
@@ -0,0 +1,205 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useAppSelector } from '@/hooks';
+import { Button, Card, Form, Select, Tooltip } from 'antd';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import Filters from './Filters';
+import { makeBeaconQuery } from '../../features/beacon/beaconQuery';
+import BeaconSearchResults from './BeaconSearchResults';
+import VariantsForm from './VariantsForm';
+
+// TODOs
+// form verification
+// example searches, either hardcoded or configurable
+// only render variants part of the form if variants are present
+// better helptext (#1691)
+
+const UI_INSTRUCTIONS = 'Search by genomic variants, clinical metadata or both.';
+const VARIANTS_HELP =
+  'Variants search requires the fields Chromosome, Variant start, Assembly ID, and at least one of Variant end or Alternate base(s).';
+const METADATA_HELP = 'Search over clinical or phenotypic properties.';
+const STARTER_FILTER = { index: 1, active: true };
+
+const WRAPPER_STYLE = {
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+};
+
+const VARIANTS_FORM_STYLE = {
+  maxWidth: '1200px',
+  display: 'flex',
+  flexWrap: 'wrap',
+};
+
+const CARD_BODY_STYLE = {
+  padding: '0 24px 5px 24px',
+};
+
+const CARD_HEAD_STYLE = {
+  border: '0',
+};
+
+const INNER_CARD_STYLE = {
+  margin: '0 5px 0 0',
+};
+
+const BUTTON_AREA_STYLE = {
+  padding: '20px 5px',
+};
+
+const BUTTON_STYLE = {
+  margin: '0 10px 0 0',
+};
+
```

Comment:
I think it might be better to move the styles to a constants folder

Reply:
not a css file?

Reply:
I am gonna look into this I get the point that css is not reused but having css in component doesnt help in understanding a component, the layout + functionality is the main thing of a component and style though important is secondary for a jsx file. Do as you like, I shall look into a standard for this.